messagebus/src/main/java/com/yahoo/messagebus/network/rpc/RPCNetwork.java

```diff
@@ -82,7 +82,7 @@ private static int getNumThreads() {
     public RPCNetwork(RPCNetworkParams params, SlobrokConfigSubscriber slobrokConfig) {
         this.slobroksConfig = slobrokConfig;
         identity = params.getIdentity();
-        orb = new Supervisor(new Transport(2));
+        orb = new Supervisor(new Transport(params.getNumNetworkThreads(), params.getOptimization() == RPCNetworkParams.Optimization.LATENCY));
```

Comment:
Consider wrapping in a `shouldEnableNoDelay(params)` method or similar to make semantics more obvious

Reply:
Fixed